spec/components/gcse_preview_component_spec.rb

```diff
@@ -41,12 +41,12 @@
 
         render_inline(described_class.new(course: course))
 
-        expect(page).to have_content("Candidates with pending GCSEs will be considered")
```

Comment:
![image](https://user-images.githubusercontent.com/470137/129372273-0cd9b936-cfd9-4b60-9acc-9b7d192459f1.png)
Should it's counterpart be updated to reflect the changes as well?

Reply:
@defong not sure what you mean here? The content means the same, but is just worded slightly differently, as the right hand side version is provider-facing and a summary of their answers.

That said, just spotted that the paragraph break between "Candidates with pending GCSEs will be considered" and "Equivalency tests will be accepted in English or maths" needs to be restored, my bad.

Reply:
(pushed a commit to fix the missing paragraph break in the summary)